tox.ini

```diff
@@ -1,12 +0,0 @@
-[tox]
-envlist = py26, py27, py33, py34, py35, pypy
-
-[testenv:py26]
-deps = unittest2
-
-[testenv:py27]
-# deps = numpy
-[testenv]
-commands = {envpython} setup.py --quiet test
-deps = setuptools
-       six
```

Comment:
Note that tox can be used locally to test several python versions at once, independent of travis or circle. I use it heavily for another project.